projects/find/find.py

```diff
@@ -0,0 +1,57 @@
+import os
+import sys
+
+"""this method is created to find all file and directory, given a specific path. Optionally, your can use -name flag
```

Comment:
nit: docstrings should be your documentation and good docstrings could generate your public docs. Always write the documentation on the code as formally as you can (capital letters, grammar, etc). Here, you should start the docstring with capital T.